Tests/Adapter/PHPCR/PHPCRAdapterTest.php

```diff
@@ -5,17 +5,16 @@
 use PHPUnit\Framework\TestCase;
 use Vich\UploaderBundle\Adapter\PHPCR\PHPCRAdapter;
 use Vich\UploaderBundle\Tests\DummyEntity;
+use Doctrine\Common\Persistence\Event\LifecycleEventArgs;
```

Comment:
The only one that can alter config is @dustin10 
Let's try with the yaml file (please open a PR for that)